src/main/java/club/thepufferfishproject/pmlinstaller/ui/frames/FrameStart.java

```diff
@@ -18,13 +22,25 @@
 public class FrameStart {
 
     public static JLabel state;
+    public static  File log = new File("pmllog.txt");
+    static {
+        try {
+            System.setOut(new PrintStream(log));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
 
     public void start() {
         System.out.println("Starting FrameStart");
 
         Dimension dim = Toolkit.getDefaultToolkit().getScreenSize();
         JButton start = new JButton("Start");
         JFrame frame;
+
+
+        System.out.println(log.getAbsolutePath());
+        System.out.println("Used space: " + log.getTotalSpace());
```

Comment:
```suggestion
        System.out.println("Used space: " + log.getTotalSpace());
```

What's this needed for?

Reply:
+1